ios/Breez Notification Service Extension/NotificationService.swift

```diff
@@ -106,6 +106,7 @@ class NotificationService: UNNotificationServiceExtension {
             }
         }
         self.paymentReceivers.removeAll(where: {$0.paymentHash == data.paymentHash})
+        try? NotificationService.breezSDK!.sync()
```

Comment:
This will trigger another sync that is already in progress. I think we should wait for that sync to complete instead of triggering another one

Reply:
That sounds like a better solution, will add a waiting mechanism for a sync. 

@roeierez  So we will have to wait for a sync event after payment succeed and not trigger on the first event which is triggered by connect?

Reply:
Yes, seems like this is the correct path.